ARCHITECTURE.md

```diff
@@ -106,6 +108,30 @@ This table keeps track of the manifests of the validators.
 | `revoked`            |Whether the manifest has been revoked.                      |
 | `seq`                |The sequence number of this manifest.                       |
 
+### `amendments_info`
+
+This table keeps track of the general information of all known amendments.
+
+| Key                  | Definition                                                 |
+|----------------------|------------------------------------------------------------|
+| `id`                 |The amendment id.                                           |
+| `name`               |The name of the amendment.                                  |
+| `rippled_version`    |The rippled version when the amendment is first enabled     |
+| `deprecated`         |Whether the amendment has been deprecated or retired        |
```

Comment:
nit:
```suggestion
| `deprecated`         |Whether the amendment has been deprecated/retired            |
```
Otherwise, it kind of reads as a boolean where the options are `deprecated` and `retired`.